bin/latest_only

```diff
@@ -2,15 +2,15 @@
 set -euo pipefail
 
 if [[ "$1" == "497" || "$1" == "496" ]]; then
-  cat <<-QUERY | psql disclosure-backend
+  cat <<-QUERY | psql ${DATABASE_NAME:-"disclosure-backend"}
```

Comment:
I would try to keep `DATABASE_NAME` specific to the Makefile and then the scripts just accept the DB as a command line argument.

Reply:
Good idea, I've addressed it subsequently.